<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proplibqteditorClass</class>
 <widget class="QMainWindow" name="proplibqteditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>proplibqteditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Config"/>
    <addaction name="actionOpen_HTTP"/>
    <addaction name="actionSave_Config"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Config">
   <property name="text">
    <string>Open Config</string>
   </property>
  </action>
  <action name="actionOpen_HTTP">
   <property name="text">
    <string>Open HTTP</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="proplibqteditor.qrc"/>
 </resources>
 <connections/>
</ui>
